src/Fantomas/CodePrinter.fs

```diff
@@ -49,6 +51,7 @@ type ASTContext =
           IsFirstTypeParam = false
           IsInsideDotGet = false
           IsInsideDotIndexed = false
+          IsInsideIfThenElseExpression = false
```

Comment:
Adding new members to `ASTContext` is a recipe for disaster.
In recent weeks I removed a couple of entries, so I won't allow any new members here.